extensions/cloud/portability-cloud-google/src/main/java/org/datatransferproject/cloud/google/GoogleCloudUtils.java

```diff
@@ -0,0 +1,46 @@
+package org.datatransferproject.cloud.google;
+
+import com.google.cloud.Timestamp;
+import com.google.cloud.datastore.Blob;
+import com.google.cloud.datastore.Entity;
+import com.google.cloud.datastore.Key;
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.ObjectOutputStream;
+import java.util.Map;
+
+public class GoogleCloudUtils {
+
```

Comment:
We should add a private no op constructor for best practices

Reply:
Done